src/javascripts/todo-list.js

```diff
@@ -138,4 +138,4 @@ class TodoList {
   }
 }
 
-export default TodoList
+return TodoList
```

Comment:
Okay. Why do we need Node to understand this? The tests are being run via PhantomJS in a browser environment, right?